tests/run_test.sh

```diff
@@ -270,7 +270,8 @@ if [[ $CPLWAV == .true. ]]; then
   if [[ $MULTIGRID = 'true' ]]; then
     atparse < ${PATHRT}/parm/ww3_multi.inp.IN > ww3_multi.inp
   else
-    atparse < ${PATHRT}/parm/ww3_shel.inp.IN > ww3_shel.inp
+    atparse < ${PATHRT}/parm/ww3_shel.nml.IN > ww3_shel.nml
```

Comment:
This is the line that made me think that we're moving away from inp files to nml files.